release_notes/index.adoc

```diff
@@ -20,3 +20,41 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+
+{product-title} provides strict backwards compatibility guarantees for all
+supported APIs (excluding alpha APIs, which may be changed without notice, and
+beta APIs, which may occasionally be changed in a non-backwards compatible
+manner). {product-title} will continue to support older clients against newer
+servers.For example, a 3.1 client will work against 3.0, 3.1, and 3.2 servers.
+
+Changes of APIs for non-security related reasons will involve, at minimum, two
+minor releases (3.1 to 3.2 to 3.3, for example) to allow older clients to
+update. Using new capabilities may require newer clients. A 3.2 server may have
+additional capabilities that a 3.1 client cannot use and a 3.2 client may have
+additional capabilities that are not supported by a 3.1 server.
+
+.Compatibility Matrix
+
+[cols="1,1,1"]
+|===
+
+|
+|*X.Y* (Client)
+|*X.Y+N* footnoteref:[versionpolicyn,Where *N* is 1 or 2.] (Client)
```

Comment:
This should read, `Where *N* is some number grater than 1` 

However with that said, the matrix only really works if your within 1-2 versions.